server/helpers/planner_helper/planner_helper_data_types.py

```diff
@@ -76,7 +76,7 @@ class ChoiceInfo(BaseModel):
 
 
 class PlanDisambiguatorInput(BaseModel):
-    selection_priority: str
+    selection_priority: Optional[str]
```

Comment:
@jkeskingvillage note in case it breaks stuff elsewhere. I have turned this to optional (to make the fast API parse on the Pydantic model work) coz in the build view it has no role.

Reply:
Yep. Build views do not read that field.